requirements/base.txt

```diff
@@ -4,29 +4,63 @@
 #
 #    pip-compile --max-rounds=100 --no-emit-index-url requirements/base.in
 #
-attrs==20.3.0             # via -r requirements/base.in
-awesome-slugify==1.6.5    # via -r requirements/base.in
-base58==2.1.0             # via -r requirements/base.in
-cached-property==1.5.2    # via -r requirements/base.in
-click-default-group==1.2.2  # via -r requirements/base.in
-click==7.1.2              # via -r requirements/base.in, click-default-group, pip-tools, rq
-colorlog==4.7.2           # via -r requirements/base.in
-numpy==1.19.4             # via -r requirements/base.in
-pendulum==2.1.2           # via -r requirements/base.in
-pip-tools==5.4.0          # via -r requirements/base.in
-plotly==4.14.1            # via -r requirements/base.in
-python-dateutil==2.8.1    # via pendulum
-python-rex==0.4           # via -r requirements/base.in
-pytz==2020.5              # via -r requirements/base.in
-pytzdata==2020.1          # via pendulum
-redis==3.5.3              # via -r requirements/base.in, rq
-regex==2020.11.13         # via awesome-slugify
-retrying==1.3.3           # via plotly
-rq==1.7.0                 # via -r requirements/base.in
-six==1.15.0               # via pip-tools, plotly, python-dateutil, python-rex, retrying
-sortedcontainers==2.3.0   # via -r requirements/base.in
-unidecode==0.4.21         # via awesome-slugify
-urllib3==1.26.3           # via -r requirements/base.in
+attrs==20.3.0
+    # via -r requirements/base.in
+awesome-slugify==1.6.5
+    # via -r requirements/base.in
+base58==2.1.0
+    # via -r requirements/base.in
+cached-property==1.5.2
+    # via -r requirements/base.in
+click-default-group==1.2.2
+    # via -r requirements/base.in
+click==7.1.2
+    # via
+    #   -r requirements/base.in
```

Comment:
The line length here isn't a problem, if possible revert them

Reply:
I'm not sure I understood what you mean. Would you like me to manually remove the newlines added by pip-tools?

Reply:
Ah, I think I understand now. I didn't add those newlines, it was automatically done by pip-tools. It might be a new format they use (which adds more information, not just the newline). 

I don't think there's an option on pip-tools to avoid this (it's been asked, but not yet implemented if I'm not wrong). I think it is better to keep it as it is, otherwise we would have to manually remove the newlines every time we compile the requirements.